src/examples/zkprogram/runtime-table/payroll.ts

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
maybe this is a little overkill but i didnt want to add something super boring :D